third_party/move/move-compiler/tests/move_check/inlining/acquires_error_msg.exp

```diff
@@ -0,0 +1,6 @@
+error[E04020]: missing acquires annotation
+  ┌─ tests/move_check/inlining/acquires_error_msg.move:7:9
+  │
+7 │         modify(); // expect error message here
+  │         ^^^^^^^^ The call acquires '0x42::test::Test', but the 'acquires' list for the current function does not contain this type. It must be present in the calling context's acquires list
```

Comment:
Is it easy to include the name of the function that's missing the acquire as well?

Reply:
Done